### PR TITLE
[FIX] repair: singleton error

### PR DESCRIPTION
**Pull request by fmdl**
**Description of the issue/feature this PR addresses:**
After this PR https://github.com/odoo/odoo/pull/111601/commits/fe2aa862413f8ef39e7b67ca902eea02960480e3, this method doesn't work with a recordset.

@Whenrow

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/repair/models/repair.py [hunk 1]
```diff
@@ -279,7 +279,7 @@ def action_repair_confirm(self):
         return True
 
     def action_repair_cancel(self):
-        if self.state == 'done':
+        if self.filtered(lambda repair: repair.state == 'done'):
```
Comment (ticodoo) [inline addons/repair/models/repair.py:282]: small code improvement

```suggestion
        if any(repair.state == 'done' for repair in self):
```

**Review by ticodoo — COMMENTED** [review]
Good catch, just a small improvement and I will merge it. Thanks!

**Comment by fmdl** [comment]
@ticodoo updated

**Comment by ticodoo** [comment]
Thanks! 

@robodoo r+

**Pull request closed without merging**